splitio/models/grammar/matchers/semver.py

```diff
@@ -0,0 +1,158 @@
+"""Semver matcher classes."""
+import logging
+
+_LOGGER = logging.getLogger(__name__)
+
+class Semver(object):
+    """Semver class."""
+
+    _METADATA_DELIMITER = "+"
+    _PRE_RELEASE_DELIMITER = "-"
+    _VALUE_DELIMITER = "."
+
+    def __init__(self, version):
+        """
+        Class Initializer
+
+        :param version: raw version as read from splitChanges response.
+        :type version: str
+        """
+        self._major = 0
+        self._minor = 0
+        self._patch = 0
+        self._pre_release = []
+        self._is_stable = False
```

Comment:
Should `_is_stable` be true by default, since `_pre_release` is an empty array by default?
Anyway, this is updated inside the `_parse` function

Reply:
yeah, it is updated in _parse.